### PR TITLE
Minor style cleanup.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: pegasus/helpers/hourofcode_helpers.rb [hunk 1]
```diff
@@ -98,17 +98,11 @@ def hoc_uri(uri)
...
-  elsif @country == 'uk'
-    return 'uk.code.org'
-  else
-    return 'code.org'
-  end
+  return 'ar.code.org' if @country == 'ar'
+  return 'br.code.org' if @country == 'br'
+  return 'ro.code.org' if @country == 'ro'
+  return 'uk.code.org' if @country == 'uk'
+  return 'code.org'
```
Comment (ashercodeorg) [inline pegasus/helpers/hourofcode_helpers.rb:105]: Is

```
if %w(ar br ro uk).include? @country
  return "#{@country}.code.org"
end
return 'code.org'
```

an improvement?

Comment (breville) [reply pegasus/helpers/hourofcode_helpers.rb:105]: I think it scales better to a big set, but for 5 cases, I think the PR's proposed fix is nice and readable.

Comment (breville) [inline pegasus/helpers/hourofcode_helpers.rb:105]: @tanyaparker probably knows this code well too.

Comment (tanyaparker) [reply pegasus/helpers/hourofcode_helpers.rb:105]: Thanks for cleaning up my old code! Will helped me previously with partner sites, can we use this here too? 

https://github.com/code-dot-org/code-dot-org/blob/d344587164a1671898b827852c72e0e892918d79/pegasus/helpers/page_helpers.rb#L13

**Pull request merged**